src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -2235,6 +2235,49 @@ paths:
       summary: Create Ceph User
       tags:
       - Cluster
+    put:
+      description: "\n        Update a ceph user with its defined capabilities.\n   \
+        \     :param user_entity: Entity to change\n        :param capabilities: List\
+        \ of capabilities to add to user_entity\n        "
+      parameters: []
+      requestBody:
```

Comment:
Looks like you didn't commit the ceph_users.py file. 

Furthermore, the description format looks off. I shouldn't start with a new line.